content/courses/mobile/mwa-deep-dive.md

```diff
@@ -518,45 +539,17 @@ npm install \
   fast-text-encoding
 ```
 
-The next step is a bit messy. We need to depend on Solana's
-`mobile-wallet-adapter-walletlib` package, which handles all of the low-level
-communication. However, this package is still in development and is not
-available through npm. From their github:
-
-> This package is still in alpha and is not production ready. However, the API
-> is stable and will not change drastically, so you can begin integration with
-> your wallet.
+We need to depend on Solana's `mobile-wallet-adapter-walletlib` package, which
+handles all of the low-level communication.
 
-However, we have extracted the package and made it available on GitHub. If
-you're interested in how we did that, take a look at the README
-[on the GitHub repo where we've made this package available](https://github.com/Unboxed-Software/mobile-wallet-adapter-walletlib)
+> A reminder that this package is still in alpha and is not production ready.
+> However, the API is stable and will not change drastically, so you can begin
+> integration with your wallet.
```

Comment:
If this is still true, you might want to use a ContentBox for this, see CONTRIBUTING.md